sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -78,6 +79,13 @@ This table shows the relationship between SDK versions and supported API version
 |5.0.0 - Latest GA release (can be installed by removing the `--pre` flag)| 3.0
 |5.1.0b3 - Latest release (beta)| 3.0, 3.1-preview.2, 3.1-preview.3
 
+Note: API version 3.1-preview.3 should be used exclusively for the Healthcare Analysis and Batch Analysis services.  When creating a Cognitive Service or Text Analytics resource for accesing these services, be sure to use flags `--sku S` and one of the following regions for the `--region` flag.
```

Comment:
My understanding is that for Healthcare you do need sku S, but not for Analyze. Is that true?

Reply:
I will double check this and follow up when I find the info.  I think this is also planned to be added to the official service docs, so I'll also see if this is something we can maybe omit from the README (see your other comment below for context).